doc/sphinx/addendum/program.rst

```diff
@@ -295,6 +295,14 @@ optional tactic is replaced by the default one if not specified.
 
    Start the proof of the next unsolved obligation.
 
+.. cmd:: Final Obligation {? of @ident } {? with @ltac_expr }
+
+   Like :cmd:`Next Obligation`, start the proof of the next unsolved
+   obligation. Additionally, after the proof is complete and the
+   automatic solver has run on any remaining obligations, it checks
+   that no obligations remain for the given :token:`ident` when
+   provided and globally in the current module when none is provided.
```

Comment:
Can you be more specific, e.g. "after the proof is complete" -> something like ```at :n:`Qed` time```
"it checks" - "it" refers to Coq or a Qed?  Maybe "the Qed checks"?